Modules/Quotes/Controllers/QuotesController.php

```
@@ -123,44 +106,37 @@ public function status(string $status = 'all', int $page = 0)
+        $items = $this->quoteItemService->getByQuoteId($quote_id);
 
         // Get tax rates
-        $taxRates      = TaxRate::all();
-        $quoteTaxRates = QuoteTaxRate::where('quote_id', $quote_id)
-            ->with('taxRate')
-            ->get();
+        $taxRates      = $this->taxRateService->getAll();
+        $quoteTaxRates = $this->quoteTaxRateService->getByQuoteId($quote_id);
 
         // Get units
         $units = Unit::all();
```

PHPMD (2.15.0)

Avoid using static access to class '\Modules\Products\Models\Unit' in method 'view'. (undefined)

(StaticAccess)